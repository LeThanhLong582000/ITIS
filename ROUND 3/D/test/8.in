7
157
41
841
81
73
251
587
