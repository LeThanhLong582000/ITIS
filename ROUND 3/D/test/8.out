919
179
6481
419
367
1597
4273
